rtt/internal/OperationInterfacePartFused.hpp

```diff
@@ -171,7 +169,7 @@ namespace RTT
                 assert( carity == collectArity() + 1 ); // check for arity functions. (this is actually a compile time assert).
                 if ( args.size() != carity ) throw wrong_number_of_args_exception(carity, args.size() );
                 // we need to ask FusedMCollectDataSource what the arg types are, based on the collect signature.
-                return new FusedMCollectDataSource<Signature>( create_sequence<typename FusedMCollectDataSource<Signature>::handle_and_arg_types >::sources(args.begin()), blocking );
+                return new FusedMCollectDataSource<Signature>( create_sequence<typename FusedMCollectDataSource<Signature>::handle_and_arg_types >::assignable(args.begin()), blocking );
```

Comment:
Not really related to this PR, but shouldn't `produceCollect()` only accept a sequence of assignable data sources? Same in [line 424](https://github.com/orocos-toolchain/rtt/pull/196/files#diff-7ea54099da3c95b19cb4c6cb852b43c1R424) for class `OperationInterfacePartFusedDS`.